content/en/blog/_posts/2024-09-17-devsecops-in-action-k8s-secrets/index.md

```diff
@@ -0,0 +1,554 @@
+---
+layout: blog
+title: "DevSecOps in Action: Kubernetes Secrets"
+slug: devsecops-in-action-k8s-secrets
+date: 2024-09-17
+author: >
+  Evgenii Frikin
+---
+
+
+The Secret is one of the basic resources that exists in the Kubernetes API. This
+kind of resource is used for different purposes and despite the name Secret,
+sensitive data is insecure to store in it. Recently considerable attention has
+been paid to
+[security improvement](/docs/tasks/administer-cluster/encrypt-data/)
+for kind of Secret resource, but Kubernetes still doesn't include appropriate
+out-of-the-box because Kubernetes philosophy carries simplicity in order not to
+compromise it engineers have to make sure Kubernetes Secret are saved
```

Comment:
I don't understand what you mean here. Can you explain?

Reply:
I wanted to say, that Kubernetes shouldn't include some solution for safe Secrets. No problem. I removed this part sentence.